packages/react/src/components/Tab/index.js

```diff
@@ -6,7 +6,7 @@
  */
 
 import * as FeatureFlags from '@carbon/feature-flags';
-import { Tab as TabNext } from '../Tabs/next/Tabs';
+import { Tab as TabNext } from '../Tabs/Tabs';
 import { default as TabClassic } from './Tab';
```

Comment:
Now if im correct on replacing using the files from the next, so any files existing in the root will be replaced (deleted, etc), then how to handle this? this is same file

unless im wrong then please correct me